PHP/PHP.sublime-syntax

```diff
@@ -9,7 +9,7 @@ file_extensions:
   - phps
   - phpt
   - phtml
-first_line_match: '^#!.*(?<!-)php[0-9]{0,1}\b'
+first_line_match: '^(#!.*(?<!-)|<\?)php[0-9]{0,1}\b'
```

Comment:
Right now this will match `#!/usr/bin/php`, `#!/usr/local/bin/php5` and `<?php`, but it will also match things like `<?php5`.

Perhaps we make the branch more explicit, and remove the lookbehind:

```
^(#!.*[^-]php[0-9]?|<\?php)\b
```